core/domain/suggestion_registry.py

```diff
@@ -1095,3 +1095,107 @@ def reject(self, reviewer_id, rejection_message):
         SuggestionTranslateContent),
     suggestion_models.SUGGESTION_TYPE_ADD_QUESTION: SuggestionAddQuestion
 }
+
+
+class ReviewerAndSuggestionCounts(python_utils.OBJECT):
```

Comment:
Can we call this `CommunityContributionSummary`?

Reply:
Isn't a summary usually a subset of condensed information which we use to display stuff in the frontend? This is more like a statistics model IMO

Reply:
@DubeySandeep @nithusha21 @BenHenning, can I get your thoughts on some naming ideas. Here’s what I have so far:
1. SuggestionAndReviewerStatsModel
2. SuggestionAndReviewerAggregateStatsModel
3. CommunityContributionStatsModel
4. ContributorDashboardStatsModel

I’m thinking of option 3 because it encompasses more than just reviewers and suggestions in case something else comes up that we want to keep track of. Option 4 also works but how likely is it that we would change the name of the dashboard? Option 3 is nice because it’s more generic. Just thought I’d get your thoughts since you all brought it up :) thanks for all your thoughts on the name and making it better!

Reply:
I like that option 3 keeps this specific to stats (which is what it contains), but 'community contribution' is pretty generic which means we will likely expand this to include various other aggregations. I think that's fine, though.

That being said, is suggestion_registry the correct place for this to live?

Reply:
Thanks @BenHenning :) I think suggestion_registry is the best place for it to live because the suggestion_registry contains all the suggestion logic in the domain layer.